llmstudio/engine/providers/provider.py

```diff
@@ -159,19 +160,14 @@ async def handle_response(
                 else request.chat_input
             ),
             "provider": self.config.id,
-            "model": (
-                request.model
-                if model and model.startswith(request.model)
-                else (model or request.model)
-            ),
-            "deployment": (
-                model
-                if model and model.startswith(request.model)
-                else (request.model if model != request.model else None)
-            ),
+            "model": model,
             "timestamp": time.time(),
-            "parameters": request.parameters.model_dump(),
+            "parameters": {
+                **request.parameters.model_dump(),
+                "deployment": request.model,
```

Comment:
this is what the client called or the value obtained on the response?

Reply:
What client called.